### PR TITLE
fix(npm): fixed npm install issues by upgrading dependency to ^5.6.0

### PR DESCRIPTION
**Pull request by omniscient**
Closing #809. Test passes and I tried it in my project. I had to delete node_modules and package.lock thought

### CODE REVIEW

**Comment by omniscient** [comment]
I did change my local dep to: `"aurelia-cli": "github:OmniscientTechnology/cli#master"` and installed cli globally to test: `npm install -g omniscienttechnology/cli#master`

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aurelia-cli",
-  "version": "0.32.0",
+  "version": "0.33.0",
```
Comment (JeroenVinke) [inline package.json]: could you undo this change? Our release process updates this automatically

Comment (omniscient) [reply package.json]: sure. one sec

**Comment by omniscient** [comment]
I re-committed in my fork but if you just wanna go ahead and simply bump the version without merging the PR, that works for me as well. Thanks for taking care of it!
